std_msgs/Header header

# offset between goal position and position computed by running FK on the IK results
geometry_msgs/Pose left_foot_ik_offset
geometry_msgs/Pose right_foot_ik_offset
geometry_msgs/Pose left_hand_ik_offset
geometry_msgs/Pose right_hand_ik_offset
